tests/containers/container_diff.pm

```diff
@@ -20,16 +20,17 @@ use containers::common;
 use containers::container_images;
 use containers::urls 'get_suse_container_urls';
 use version_utils qw(is_sle get_os_release);
+use containers::runtime;
 
 sub run {
     my ($self) = @_;
     $self->select_serial_terminal;
-
+    my $docker = containers::runtime::docker->new();
```

Comment:
```suggestion
    my $runtime = containers::runtime::docker->new();
```
We know it's docker, but let's use the generic name.

Reply:
but in this module make more sense the name of the variable to reflect the actual runtime. that is because the module always use docker.